app/src/main/java/com/nlinterface/activities/MainActivity.kt

```diff
@@ -64,6 +65,10 @@ class MainActivity : AppCompatActivity() {
         configureUI()
         configureTTS()
         configureSTT()
+
+        verifyCameraPermissions()
+        val serviceIntent = Intent(this, ConstantScanning()::class.java)
+        startService(serviceIntent)
```

Comment:
Bin mir auch nicht sicher, ob das Barcode-Scannen aktuell permanent als Service laufen sollte...das zieht ziemlich Akku, wenn die Kamera die ganze Zeit an ist. Vielleicht sollte es erst mal in eine separate Activity ausgelagert werden.